pkg/app/piped/notifier/slack.go

```diff
@@ -51,14 +53,30 @@ type slack struct {
 }
 
 func newSlackSender(name string, cfg config.NotificationReceiverSlack, webURL string, logger *zap.Logger) *slack {
+	var oauthtoken string
+	if cfg.OAuthTokenData != "" {
+		a, err := base64.StdEncoding.DecodeString(cfg.OAuthTokenData)
+		if err != nil {
+			logger.Error(fmt.Sprintf("failed to decode the oauth token data: %v", err))
+			return nil
+		}
+		oauthtoken = string(a)
+	}
+	if cfg.OAuthTokenFile != "" {
+		a, err := os.ReadFile(cfg.OAuthTokenFile)
+		if err != nil {
+			logger.Error(fmt.Sprintf("failed to read the oauth token file: %v", err))
```

Comment:
```suggestion
			return nil, fmt.Sprintf("failed to read the oauth token file: %w", err)
```

Reply:
I think I should use Errorf since we added error signature on this func